include/SFML/Graphics/Shape.hpp

```diff
@@ -256,12 +256,6 @@ class SFML_GRAPHICS_API Shape : public Drawable, public Transformable
     FloatRect getGlobalBounds() const;
 
 protected:
-    ////////////////////////////////////////////////////////////
-    /// \brief Default constructor
-    ///
-    ////////////////////////////////////////////////////////////
-    Shape();
```

Comment:
There is no need to hide this constructor. This class already contains some pure virtual functions making it not constructible by default. The `sf::Shape` tests already assert that this type is not constructible.